src/screens/ErrorPage.js

```diff
@@ -0,0 +1,28 @@
+import { Button } from "../components";
+import { render } from "../shared/dom";
+
+function onClickHandler() {
+    this.$routes.push('Home');
```

Comment:
We have dedicated "redirect" function in project for such a case

Reply:
I'll change that